src/app/table/column-data.module.ts

```diff
@@ -0,0 +1,21 @@
+export class ColumnData {
```

Comment:
Ova klasa ti nije potrebna iz razloga sto je neodrziva. Ako se nova kolona doda to bi trebalo izmjeniti samo u jednom fajlu - glavnoj komponenti (niz naziva kolona).